frontend/src/css/lib/tree.js

```diff
@@ -0,0 +1,520 @@
+/*
```

Comment:
Why this file is under **css** folder? It would probably be more appropriate to put this file under **helpers**, since all other useful _js_ files are there. What do you think?

Reply:
Wherever it is, it should be under a `/lib` folder so it's clear we didn't write it.

Reply:
I agree with @caleybrock. The only thing here, is that this file is under **css** which is little weird.

Reply:
I can do helpers/lib?

Reply:
@MichaelCherry I'm fine with that.

Reply:
Moved in 93a33fa